style/js/shield_defs.js

```diff
@@ -484,8 +523,9 @@ export function loadShields(shieldImages) {
     },
   };
 
-  shields["US:PA:Business"] = banneredShield(shields["US:PA"], ["BUS"]);
   shields["US:PA:Truck"] = banneredShield(shields["US:PA"], ["TRK"]);
+  shields["US:PA:Business"] = banneredShield(shields["US:PA"], ["BUS"]);
```

Comment:
US:PA:Business in Wilkes-Barre, PA:
![image](https://user-images.githubusercontent.com/3254090/154863798-260d1106-35a2-4f88-aede-b353f245a405.png)